File: Java/hashmap.java
import java.util.HashMap;

public class Main {
    public static void main(String[] args) {
        HashMap<String, Object> hashmap = new HashMap<>();
        hashmap.put("name", "Alice");
        hashmap.put("age", 25);
        hashmap.put("city", "Moscow");
        
        System.out.println(hashmap.get("name"));  // Alice
        hashmap.put("age", 26);  // Изменение значения
    }
}

File: C++/hashmap.cpp
#include <iostream>
#include <unordered_map>
#include <string>

int main() {
    std::unordered_map<std::string, std::string> hashmap;
    hashmap["name"] = "Alice";
    hashmap["city"] = "Moscow";
    
    std::cout << hashmap["name"] << std::endl;  // Alice
    hashmap["city"] = "Saint Petersburg";
    
    return 0;
}
